flow/raster_cache.cc

```diff
@@ -92,13 +92,14 @@ static bool IsPictureWorthRasterizing(SkPicture* picture,
   return picture->approximateOpCount() > 5;
 }
 
+// TODO(aaclarke): Refactor to FunctionRef.
```

Comment:
The TODO implies FunctionRefs are going to be imminently added. This is not the case pending further discussion about its merits.

Reply:
made the notes more agnostic, the research is still valuable if we decide to use functionref or something from std later.